DateRangeBox: Add Getting Started with DateRangeBox guide (#5041)
DevExpress/devextreme-documentation: concepts/05 UI Components/DateRangeBox/00 Getting Started with DateRangeBox/25 Handle the Value Change Event.md

> To handle a date range change, you can implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#onValueChanged') function. In this tutorial, the callback function logs the previous and current range of dates.

Reviewer: ```suggestion
To respond to a date range change, handle the [onValueChanged](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#onValueChanged') event. In this tutorial, the callback function logs the previous and current date ranges.
```

    import { DxDateRangeBox } from 'devextreme-vue/date-range-box';

    export default {
        // ...
        methods: {
            onValueChanged(e) {
                console.log(e.value);
                console.log(e.previousValue);
            }
        }
    }
    </script>

    <!-- tab: App.vue (Composition API) -->
    <template>
        <DxDateRangeBox ...
            @value-changed="onValueChanged"
        />
    </template>

    <script setup>
    import 'devextreme/dist/css/dx.light.css';

    import { DxDateRangeBox } from 'devextreme-vue/date-range-box';

    const onValueChanged = (e) => {
        console.log(e.value);
        console.log(e.previousValue);
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';

    import 'devextreme/dist/css/dx.light.css';

    import { DateRangeBox } from 'devextreme-react/date-range-box';

    const onValueChanged = (e) => {
        console.log(e.value);
        console.log(e.previousValue);  
    } 

    export default function App() { 
        return (
            <DateRangeBox ...
                onValueChanged={onValueChanged}
            />
        );
    }

---

> You have now completed DateRangeBox basic configuration. For additional information on this UI component, explore the following resources:

Reviewer 2: ```suggestion
You have now completed DateRangeBox basic configuration. For additional information on this UI component, see the following link:
```

> You have now configured the basic DateRangeBox features. For more details on this UI component, explore the following resources:

Reviewer: ```suggestion
You have now completed DateRangeBox basic configuration. For additional information on this UI component, explore the following resources:
```
